webportal/webapp/src/app/account/loginsync/components/loginsync.component.html

```diff
@@ -1,24 +1,47 @@
 <div class="container">
-  <mat-card>
     <app-loader></app-loader>
-    <div>
-      <app-alert [showAlert]="!linkSuccessful" [text]="errorMessage"></app-alert>
-      <form class="usa-form signin" (ngSubmit)="onSubmit()">
-        <fieldset class="usa-fieldset">
-          <legend class="usa-legend">One Time Sign In</legend>
-          <label for="username">Username</label>
-          <input class="usa-input" id="username" name="username" [(ngModel)]="username" autocapitalize="off" autocorrect="off">
-  
-          <label for="password-sign-in">Password</label>
-          <input class="usa-input" id="password-sign-in" name="password" [(ngModel)]="password" type="password" appTogglePassword>
-          <input class="usa-button link-submit" type="submit" value="Sign in">
+    <div class="grid-row margin-top-3">
```

Comment:
I merged the regular sync account screen with the password reset screen. Most of the functionality is shared, and they seem to fit pretty well together. The component + view don't seem to be too large...